R/get_-functions.R

```diff
@@ -108,7 +115,7 @@ get_type <- function(x) {
   }
   assert_numeric(as.vector(x))
   if (all(is.na(as.vector(x)))) {
-    stop("Can't get type: all values of are NA")
+    cli_abort("Can't get type: all values of are NA")
```

Comment:
```suggestion
    cli_abort("Can't get type: all values of are {.val {NA}}.")
```

Reply:
Is it supposed to read "all values ~of~ are NA"? There's something missing currently.